src/component/actionbar/index.tsx

```diff
@@ -273,6 +273,8 @@ function ActionBar(props: ActionBarProps) {
                   {...attributes.popper}
                 >
                   <Picker
+                    showPreview={false}
+                    showSkinTones={false}
```

Comment:
This is a built-in way to prevent the emoji-mart bottom section which was originally done in https://github.com/venturemark/webclient/pull/422